app/src/main/java/com/github/se/gatherspot/ui/ChatUI.kt

```diff
@@ -0,0 +1,35 @@
+package com.github.se.gatherspot.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import com.github.se.gatherspot.ProfileFirebaseConnection
+import com.github.se.gatherspot.model.Profile
+import com.github.se.gatherspot.model.chat.Chat
+import com.github.se.gatherspot.model.event.Event
+import com.github.se.gatherspot.ui.navigation.NavigationActions
+import kotlinx.coroutines.runBlocking
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ChatUI(chat: Chat, nav: NavigationActions) {
+
+  var profiles: List<Profile?>? = null
+  var event: Event? = null
+  runBlocking {
```

Comment:
Idealy this part should not be in EventUI since it's rather functionality than displaying (MVVM model)